lua/bufferline/bbye.lua

```diff
@@ -111,24 +119,20 @@ function bbye.delete(action, force, buffer, mods)
 
   -- For cases where adding buffers causes new windows to appear or hiding some
   -- causes windows to disappear and thereby decrement, loop backwards.
-  local window_ids = vim.api.nvim_list_wins()
-  local window_ids_reversed = {}
-  while #window_ids_reversed < #window_ids do
-    window_ids_reversed[#window_ids_reversed + 1] = window_ids[#window_ids - #window_ids_reversed]
-  end
+  local window_numbers = vim.api.nvim_list_wins()
```

Comment:
Here, `window_ids` is the correct name because both window ID and window number concepts exist in neovim, and `nvim_list_wins()` returns window IDs.